Project-Aurora/Project-Aurora/Devices/Device.cs

```diff
@@ -1301,6 +1301,96 @@ public enum DeviceKeys
         [Description("Calculator")]
         CALC = 216,
 
+        /// <summary>
+        /// Peripheral Light 1
+        /// </summary>
+        [Description("Peripheral Light 1")]
+        Peripheral_Light1 = 217,
+
+        /// <summary>
+        /// Peripheral Light 2
+        /// </summary>
+        [Description("Peripheral Light 2")]
+        Peripheral_Light2 = 218,
+
+        /// <summary>
+        /// Peripheral Light 3
+        /// </summary>
+        [Description("Peripheral Light 3")]
+        Peripheral_Light3 = 219,
+
+        /// <summary>
+        /// Peripheral Light 4
+        /// </summary>
+        [Description("Peripheral Light 4")]
+        Peripheral_Light4 = 220,
+
+        /// <summary>
+        /// Peripheral Light 5
+        /// </summary>
+        [Description("Peripheral Light 5")]
+        Peripheral_Light5 = 221,
+
+        /// <summary>
+        /// Peripheral Light 6
+        /// </summary>
+        [Description("Peripheral Light 6")]
+        Peripheral_Light6 = 222,
+
+        /// <summary>
+        /// Peripheral Light 7
+        /// </summary>
+        [Description("Peripheral Light 7")]
+        Peripheral_Light7 = 223,
+
+        /// <summary>
+        /// Peripheral Light 8
+        /// </summary>
+        [Description("Peripheral Light 8")]
+        Peripheral_Light8 = 224,
+
+        /// <summary>
+        /// Peripheral Light 9
+        /// </summary>
+        [Description("Peripheral Light 9")]
+        Peripheral_Light9 = 225,
+
+        /// <summary>
+        /// Peripheral Light 10
+        /// </summary>
+        [Description("Peripheral Light 10")]
+        Peripheral_Light10 = 226,
+
+        /// <summary>
+        /// Peripheral Light 11
+        /// </summary>
+        [Description("Peripheral Light 11")]
+        Peripheral_Light11 = 227,
+
+        /// <summary>
+        /// Peripheral Light 12
+        /// </summary>
+        [Description("Peripheral Light 12")]
+        Peripheral_Light12 = 228,
+
+        /// <summary>
+        /// Peripheral Light 13
+        /// </summary>
+        [Description("Peripheral Light 13")]
+        Peripheral_Light13 = 229,
+
+        /// <summary>
+        /// Peripheral Light 14
+        /// </summary>
+        [Description("Peripheral Light 14")]
+        Peripheral_Light14 = 230,
+
+        /// <summary>
+        /// Logo #4
+        /// </summary>
+        [Description("Brand Logo #4")]
+        LOGO4 = 231,
```

Comment:
Why are you created the logo4 instead of using the DeviceKeys.Peripheral_Logo?

Reply:
If we use peripheral logo then it would conflict with [RazerDevice.cs:197](https://github.com/antonpup/Aurora/pull/2119/files#diff-f4a61ccdb54400389966f74025f4c84cR197)